nais-q-sbs.yaml

```diff
@@ -11,6 +11,8 @@ spec:
   ingresses:
     - https://www-{{namespace}}.nav.no/frontendlogger
     - https://frontendlogger-{{namespace}}.nais.oera-q.local
+    - https://tjenester-{{namespace}}.nav.no/frontendlogger
+    - https://arbeidsgiver-q.nav.no/frontendlogger
```

Comment:
Hvorfor trengs denne egentlig? Trodde det var bestemt at subdomener ikke skulle brukes? 😕 

Og skal endringen ikke være med til Prod? 😕

Reply:
Den avgjørelsen har ikke nådd oss i PO arbeidsgiver i alle fall 😅 
Frontendlogger ble eksponert hos oss da den var på naisd, og vi har en del metrikker som nå ikke kommer gjennom.
Vi har stort sett gått over i å sende metrikker utenom frontendlogger, men for i alle fall én applikasjon har vi ikke kommet så langt. 

Ang prod: Jeg trodde flyten var branch -> dev/testmiljø ->master/prod. Hvordan gjør jeg det ellers? En PR til dev og en til master?

Reply:
Mulig områder skal ha subdomener da. :/ 
Bare jeg som er skeptisk til å legge til for mye her. Noen kjører appene sine med subdomene (e.g mininnboks.nav.no), og da kommer den listen der til å bli evig lang om alle skal ha appen sin der også. :/ 

Angående branch-flyten så vil dev deploye til preprod, og master til prod, men jeg stusset på at det ikke var noen referanser til `arbeidsgiver.nav.no`. Ser nå at det er gjort endringer i `nais-sbs.yaml`, men at den refererer til `arbeidsgiver-q.nav.no`. Så det må nok endres. :)

Reply:
Dette funker ikke for alle test miljøene, burde stå

`https://arbeidsgiver-{{namespace}}.nav.no/frontendlogger`

Reply:
Om man bare bruker ett q-miljø, og urlen er `appnavn-q.nav.no` så vil det ikke fungere å bruke `namespace` siden den variablen blir satt til `q0` i denne appen. Så for preprod-urlen så vil nok være riktig med `arbeidsgiver-q.nav.no` her

Reply:
Sant. Så denne blir litt snodig hvis vi deployer den til flere test miljø, men atm så deployer vi kunn til q0

Reply:
Så da er det vel greit da

Reply:
Yeah, det vil bli litt rart. Ikke helt sikker på hvordan det vil bli egentlig. :/ 
E.g podder fra q0, q1 og q6 har alle arbeidsgiver-q som ingress. Vil det da bare bli lastbalansert mellom de ulike namespacene? Vil ihvertfall bli hårete å se på loggene for team-arbeidsgiver siden de ikke vet hvilken namespace loggene havner i.

Reply:
Bra vi bare søtter q0 enn så lenge, så får vi evt fjerne denne url-en om vi ønkser frontendloggeren i andre miljø ?

Reply:
Testlenken var feil ja. Jeg tror jeg ble forvirret av "https://frontendlogger.nais.oera-q.local" som ligger i nais-sbs.yaml. Skal den være adeo.no?

Reply:
God poeng med namespace, det kan bli litt merkelig.. Hvordan fungerte dette på naisd-versjonen?

Reply:
funket fordi man fikk variabler fra fasit

Reply:
hmm ja  "https://frontendlogger.nais.oera-q.local" feil. Fjerner du den i samme slengen ?